### PR TITLE
refactor(frontend): extract component from token card

### PR DESCRIPTION
**Pull request by AntonioVentilii-DFINITY**
# Motivation

We would like to re-use the component TokenCard but without the hyperlink.

# Changes

- Rename `TokenCard` component.
- Extract sub-component.

### CODE REVIEW

File: src/frontend/src/lib/components/tokens/TokenCardWithUrl.svelte [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+	let url: string;
+	$: url = transactionsUrl({ token });
+</script>
+
+<div class="flex gap-3 sm:gap-8 mb-6">
+	<a
+		class="no-underline flex-1"
+		href={url}
+		aria-label={`Open the list of ${token.symbol} transactions`}
```
Comment (peterpeterparker) [inline src/frontend/src/lib/components/tokens/TokenCardWithUrl.svelte:16]: Unrelated. We need an i18n key here.

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

File: src/frontend/src/lib/components/tokens/TokenCardWithUrl.svelte (hunk 1 above)
Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/components/tokens/TokenCardWithUrl.svelte:16]: right!

Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/components/tokens/TokenCardWithUrl.svelte:16]: Created PR #1863 for that

**Pull request merged**